src/Http/Concerns/CustomUrls.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Http\Concerns;
+
+trait CustomUrls
```

Comment:
The trait and the interface also enforces the implementation of `shortenToAsync`. I would only enforce it with the interface and use the trait for the default implementations for `shortenTo` and rename the trait to `HasCustomUrls` or something similar to not conflict with the name of the interface.

Reply:
The reason I did that is because otherwise the trait references a method that might not exist :)